frontend/README.md

```diff
@@ -0,0 +1,27 @@
+# getline-frontend
+
+> This is the frontend app for the Getline Distributed P2P Lending Platform
+
+## Build Setup
+
+``` bash
+# install dependencies
+npm install
```

Comment:
Mind if we use/recommend yarn instead of npm?

Reply:
sure, no problem, changed to yarn